k8s/openebs-config.yaml

```diff
@@ -47,6 +47,28 @@
 #  - a directory on host-os, 
 #  - a directory on mounted-disk, 
 #
+# Define a storage pool for launching the replicas using
+#  local directory or mounted directory on kubernetes nodes
+#  
+# The following is an example of how a storagepool can be 
+# confgured with any other directory to be used for persisting 
+# the storage. 
+#
+# Note: OpenEBS uses /var/openebs as default directory, even if 
+# the following default storagepool is not added to the k8s cluster.
+#
+apiVersion: openebs.io/v1alpha1
+kind: StoragePool
+metadata:
+  name: default
```

Comment:
I have had always this question in mind w.r.t. these yamls present here.
Are we expecting the user to download & use these directly ?
Which environment will the user be interested to install these yamls ?
The namespace is hardcoded to default. Does it mean, the user is expected to modify the yamls ?
Is the user expected to modify manually or are we providing some tools ?

Reply:
Till we come up with an operator (code) we will need to manually download these yamls, customize them and apply in the kubernetes cluster. Another option is to use helm, which actually packages all these yaml files into a chart, with parameters to change them during helm install. 

The yaml files are really for those who would like to dig down and see the different pieces that are involved in enabling the openebs.